config.js

```diff
@@ -125,6 +125,9 @@ var config = {
     // Disables the reactions moderation feature.
     // disableReactionsModeration: false,
 
+    // Disables the reactions in chat feature.
+    disableReactionsInChat: true,
+
```

Comment:
The default value should be to keep current behavior and the config to be commented in config.js.